### ISSUE TITLE
How to get probability for each output tokens of FTBART model when inference?

### ISSUE THREAD

**Issue by quancq** [NVIDIA/FasterTransformer#621]
I get output scores for each output token of HuggingFace BART model by: ``model.generate(input_ids, output_scores=True)``. I calculate probability by softmax.
I follow [example](https://github.com/NVIDIA/FasterTransformer/blob/main/examples/pytorch/bart/bart.ipynb). I set ``is_return_output_log_probs=True``, but i dont know how to get probability from log_prob. I use numpy.exp, but almost result equal 1.
Can you help me to get probability for each output tokens of FTBART model?
Thanks a lot.
